plugin-examples/src/main/java/com/hotels/styx/GeneralExamplePlugin.java

```diff
@@ -32,15 +32,15 @@
  * You can change it in whatever way you like (provided it still implements the Plugin interface),
  * and should rename it to something relevant to your project.
  */
-public class ExamplePlugin implements Plugin {
-    private final ExamplePluginConfig config;
+public class GeneralExamplePlugin implements Plugin {
```

Comment:
Perhaps instead, we should name it `ModifyHeadersExamplePlugin`, to clarify its behaviour (please not we would also need to rename the config, factory and test classes accordingly).